src/main/plugins/org.dita.pdf2/cfg/common/vars/ar.xml

```diff
@@ -328,16 +328,46 @@ See the accompanying license.txt file for applicable licenses.
 
     <!-- The header that appears on the even-numbered glossary pages. -->
     <variable id="Glossary even header"><param ref-name="pagenum"/> | <param ref-name="heading"/> | <param ref-name="prodname"/></variable>
-     
+
      <!-- The heading string to put at the top of the Glossary -->
      <variable id="Glossary">قاموس المصطلحات</variable>
-     
+
      <!-- The heading string to put at the top of the List of Tables -->
      <variable id="List of Tables">الجداول</variable>
-     
+
      <!-- The heading string to put at the top of the List of Figures -->
      <variable id="List of Figures">الأشكال</variable>
 
+     <!-- The heading string to put at the top of the List of Changes -->
+     <variable id="List of Changes"></variable>
```

Comment:
@robander I think we should have a coding guideline for a case like this. What to do when adding a new variable string. Do you just add the English source text there and add e.g. a comment "FIXME: translate" or something to show that the content hasn't been translated yet?